src/main/java/pl/mantiscrab/budgetr/domain/user/User.java

```diff
@@ -0,0 +1,24 @@
+package pl.mantiscrab.budgetr.domain.user;
+
+import lombok.*;
+
+import javax.persistence.*;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Entity
+@Table(name = "budgetr_users")
+class User {
```

Comment:
The User in here is a bit confusing from my perspective, but for now i don't have any better idea how to name it so keep it as it is